README.md

```diff
@@ -3,42 +3,45 @@
 ## Prepare release
 
 **Clone repository**
-
 ```
 git clone https://github.com/wazuh/wazuh-bosh
 cd wazuh-bosh
 ```
 
-**Download blobs from Git LFS (Ubuntu/Debian)**
-
+**Install Git LFS (Ubuntu/Debian)**
 ```
 curl -s https://packagecloud.io/install/repositories/github/git-lfs/script.deb.sh | sudo bash
 sudo apt-get install git-lfs
-git lfs install
-git lfs pull
 ```
 
-**Upload blob**
+**Install Git LFS (MacOS)**
+```
+brew install git-lfs
+```
 
-Upload the blobs to the blob store.
+**Download blobs from Git LFS**
```

Comment:
I think *Download blobs from `wazuh-bosh` repository using Git LFS*  woulld be a better title for this

Reply:
I've modified it following the suggestion, please take a look